lib/jobs/ServiceInstanceUpdateJob.js

```diff
@@ -211,7 +212,8 @@ class ServiceInstanceUpdateJob extends BaseJob {
             break;
           }
         }
-        logger.info('Initial run status - ', initialJobRunStatus);
+        logger.info('Initial run status - ', _.pick(initialJobRunStatus, 'data', 'response', 'statusCode', 'statusMessage', 'interval'));
+        logger.info('Last run status - ', _.pick(lastRunList[0], 'data', 'response', 'statusCode', 'statusMessage', 'interval'));
```

Comment:
In above two I think not mandatory to add SI update job  as data already contains the info.

Reply:
Did not understand the above comment. What in the above logs are you referring as SI Update Job?

Reply:
Ignore. I meant the above logger is perfect :)